index.js

```diff
@@ -652,7 +652,7 @@ function generateIndex (docIndexFile) {
 }
 
 function makeDataDir () {
-	mkdirp.sync(dataDir);
+	mkdirp.mkdirpSync(dataDir);
```

Comment:
Note: After updating to version 3.0.0 mkdirp.sync(dataDir) would generate an error `mkdirp.sync is not a function`